
From: Jesse Barnes <jbarnes@engr.sgi.com>

And apparently it's broken on NUMA.  I couldn't find 
generic_file_get/set_policy in my tree, which builds with CONFIG_NUMA 
enabled.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/mm/filemap.c |    4 ----
 1 files changed, 4 deletions(-)

diff -puN mm/filemap.c~add-page-becoming-writable-notification-build-fix mm/filemap.c
--- 25/mm/filemap.c~add-page-becoming-writable-notification-build-fix	2005-01-10 21:15:04.986695488 -0800
+++ 25-akpm/mm/filemap.c	2005-01-10 21:15:04.990694880 -0800
@@ -1520,10 +1520,6 @@ struct vm_operations_struct generic_file
 	.nopage		= filemap_nopage,
 	.populate	= filemap_populate,
 	.page_mkwrite	= filemap_page_mkwrite,
-#ifdef CONFIG_NUMA
-	.set_policy     = generic_file_set_policy,
-	.get_policy     = generic_file_get_policy,
-#endif
 };
 
 /* This is used for a general mmap of a disk file */
_
